server.js

```diff
@@ -2,22 +2,87 @@ const express = require('express');
 const bodyParser = require('body-parser');
 const app = express();
 
+const menu = require('./menu.js');
+const {getMenu, getMenuItem} =menu();
+
+const order = require('./order.js');
+const {createNewOrder, changeOrderItem, getOrder, deleteFoodItem, deleteOrder, orderList} = order();
+
+
 app.use(bodyParser.json());
 app.use('/static', express.static('static'));
 app.set('view engine', 'hbs');
 
-const menu = {
-  1: {
-    id: 1,
-    name: "Strawberry cheesecake",
-    price: 6
-  }
-};
 
-app.get('/', function(req, res){
+// #1
+app.get('/', (req, res) => {
   res.render('index');
 });
 
-app.listen(8080, function(){
+// #2
+app.get('/menu', (req, res) => {
+  res.json(getMenu())
+});
+
+// #3
+app.get('/menu/:id', (req, res) => {
+  const result = getMenuItem(req.params.id);
+  if (result) {
+    res.json(result);
+  } else {
+    res.status(404).send('The food item with the given ID was not found');
```

Comment:
It would better the send the error message as JSON so that it can be read by fetch the same way as normal result. Otherwise, the front end will throw an error when it tries to convert a string to object.